.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
     - angular/node_modules
 
 env:
-  - DB=postgresql CXX=g++-4.8
+  - DB=postgresql CXX=g++-4.8 CANONICAL_HOST=loomio.example.org
```

Comment:
I don't, heh. I modified the EmailParams class slightly to allow it to be tested a little more easily.